src/main/java/net/mcreator/ui/dialogs/JavaModelAnimationEditorDialog.java

```diff
@@ -100,7 +53,10 @@ public static String openAnimationEditorDialog(MCreator mcreator, String modelSo
 		Map<String, JComboBox<String>> animations = new HashMap<>();
 
 		for (String part : vc) {
-			JComboBox<String> box = new JComboBox<>(java_model_animations.keySet().toArray(new String[0]));
+			List<String> types = new ArrayList<>(BuiltInEntityAnimations.getAllTexts());
+			types.sort(String::compareTo);
+			JComboBox<String> box = new JComboBox<>(types.toArray(new String[0]));
+			box.setSelectedItem(L10N.t("animations.entities.no_anim"));

```

Comment:
This dropdown here should really store by key, or not use translations. Using translated strings as a key is not a good practise. That's why all (most, except those who store by key) dropdowns in MCreator are not translated.